src/lang/fa.rs

```diff
@@ -483,7 +483,7 @@ pub static ref T: std::collections::HashMap<&'static str, &'static str> =
         ("System Sound", "صدای سیستم"),
         ("Default", "پیش فرض"),
         ("New RDP", "ریموت جدید"),
-        ("Fingerprint", "اثر انگشت"),
+        ("Fingerprint", "\n اثر انگشت"),
```

Comment:
`\n`?

Reply:
![image](https://github.com/rustdesk/rustdesk/assets/19924924/a542f24d-c788-4fa5-b7a8-4780d7677ab9)
need `\n ` to show correct.